plugin/yt_dlp_plugins/extractor/getpot_bgutil.py

```diff
@@ -20,65 +21,71 @@ def _validate_get_pot(self, client: str, ydl: YoutubeDL, visitor_data=None, data
 
     def _get_pot(self, client: str, ydl: YoutubeDL, visitor_data=None, data_sync_id=None, player_url=None, **kwargs) -> str:
         generate_pot_script_path = ydl.get_info_extractor('Youtube')._configuration_arg('getpot_bgutil_script', [None], casesense=True)[0]
+        http_base_url = ydl.get_info_extractor('Youtube')._configuration_arg('getpot_bgutil_baseurl', ['http://127.0.0.1:4416'], casesense=True)[0]
```

Comment:
Are there any cases where the server wouldn't be running on the loopback address? 

My understanding is that the PO token has to be generated on the same IP address as the original request, so it should always be on 127.0.0.1? In that case, the only variable thing is probably the port number?

Reply:
Could be running on a different IP on local network, or use docker container hostname/IP (likely be a common scenario if yt-dlp is also running in a docker container).